client/services/FeedbackGeneratorService.js

```diff
@@ -52,6 +52,21 @@ tie.factory('FeedbackGeneratorService', [
               ].join(''), false);
             }
           }
+          // TODO(eyurko): Make this fancier. Add in more data points.
```

Comment:
Please elaborate on what "fancier" means. TODOs should, in principle, have enough information to be tackle-able by anyone.